src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,18 @@
+package seedu.addressbook.data.person;
+
+public class Contact {
+	
+	private boolean isPrivate;
+	
+	public Contact() {
+		
+	}
+	
+	public Contact(boolean status) {
```

Comment:
You can pass more information to the super class constructor as mentioned above Address.java class